### PR TITLE
Update file selector to use common-header subscription-status component

### PR DESCRIPTION
**Pull request by stulees**
Fixes https://github.com/Rise-Vision/widget-video/issues/251

### CODE REVIEW

**Comment by stulees** [comment]
@alex-deaconu Do you mind reviewing this? Santiago is out today, thanks!

File: bower.json [hunk 1]
```diff
@@ -31,13 +31,13 @@
...
+    "angular-ui-router": "^0.2.18",
     "angular-ui-tinymce": "0.0.18",
     "bootstrap-form-components": "https://github.com/Rise-Vision/bootstrap-form-components.git",
     "bootstrap-sass-official": "~3.2.0",
     "component-storage-selector": "https://github.com/Rise-Vision/component-storage-selector.git",
-    "component-subscription-status": "https://github.com/Rise-Vision/component-subscription-status.git",
+    "common-header": "https://github.com/Rise-Vision/common-header.git",
     "rv-angular-bootstrap-colorpicker": "https://github.com/Rise-Vision/angular-bootstrap-colorpicker.git",
     "rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git",
     "rv-common-style": "https://github.com/Rise-Vision/common-style.git",
```
Comment (alex-deaconu) [inline bower.json]: @stulees you can actually remove the common-i18n and common-style references here; those repos are no longer used.

Also, in Apps we simplify the dependencies and necessary resolutions by just allowing Common-Header to define the Angular version and other library versions (including jquery, bootstrap, angular plugins, etc). Just a suggestion for keeping the bower.json file clean.

Thanks!

**Comment by alex-deaconu** [comment]
@stulees looks good, just one cleanup suggestion in the code. Thanks!

**Comment by stulees** [comment]
@alex-deaconu Good catch on removing common-i18n and common-style references in entirety. I've now done so and also done a bit of additional cleanup of bower, although still needed a few resolutions because of other bower dependencies. Thanks!

**Pull request merged**